dapp/src/components/buySell/BalanceHeader.js

```diff
@@ -220,10 +122,10 @@ const BalanceHeader = () => {
           </div>
           <div className="ousd-value-holder d-flex flex-column align-items-start justify-content-center">
             <div className="light-grey-label d-flex">
-              {fbt('Estimated OUSD Balance', 'Estimated OUSD Balance')}
+              {fbt('OUSD Balance', 'OUSD Balance')}
             </div>
             <div className={`ousd-value ${balanceEmphasised ? 'big' : ''}`}>
-              {typeof displayedBalanceNum === 'number' &&
+              {typeof parseFloat(displayedBalance) === 'number' && !isNaN(displayedBalance) &&
```

Comment:
@ZaHuPro sorry for a late comment on this one. Have just tested this code on my end. I have experienced a couple of issue, I hope you could investigate: 
1. in the above code the `!isNaN(displayedBalance)` should probably be `!isNaN(parseFloat displayedBalance))`
2. When I use account that has 0 OUSD balance the widget is stuck at a loading state showing just `--.----`
3. When I switch from 1 account that has OUSD to another account that has 0 OUSD in Metamask. The widget shows the balance from the first account instead of showing 0 OUSD. (This could be related to the 2nd issue)

Reply:
@sparrowDom  I will look at it

Reply:
@sparrowDom  I have made these changes in this PR https://github.com/OriginProtocol/origin-dollar/pull/403